extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -261,7 +268,11 @@ export class Bind {
 
     const expression = args[RAW_OBJECT_ARGS_KEY];
     if (expression) {
-      // Signal on first mutation.
+      // Increment bindings limit by 500 on each invocation to a max of 2000.
+      this.maxNumberOfBindings_ = Math.min(2000,
+          Math.max(1000, this.maxNumberOfBindings_ + 500));
```

Comment:
The `Math.max` should be unnecessary here.

Reply:
Ah, I used to have this as `Math.max(1000, 1000 + this.numberOfInvocations_ * 500)`. I'll clean this up soon.